pkg/controller/registry/resolver/operators.go

```diff
@@ -238,18 +238,14 @@ type Operator struct {
 
 var _ OperatorSurface = &Operator{}
 
-func NewOperatorFromBundle(bundle *api.Bundle, replaces string, startingCSV string, sourceKey CatalogKey) (*Operator, error) {
+func NewOperatorFromBundle(bundle *api.Bundle, startingCSV string, sourceKey CatalogKey) (*Operator, error) {
```

Comment:
Just asking because I don't know how this works very well:\
since the replaces field in the db is an int64 and what is used here is a string that is the equivalent of the replaces field in the csv - does the `o.Identifier()` extract that replaces csv name from the database/query?

Reply:
The values from `replaces` in the registry isn't seen by olm at all. 

OLM queries for a replacement with `FindReplacement`, and uses whatever the registry returns as the value for the "replaces" field that it puts in the CSV.